examples/scim-patch/main.go

```diff
@@ -0,0 +1,127 @@
+package main
```

Comment:
Use godoc examples to make it discoverable https://go.dev/blog/examples

make doc to preview